tests/scenario/test_start_statuses.py

```diff
@@ -53,13 +50,13 @@ def patch_all(substrate, placeholder_cfg_path):
             yield
 
     else:
-        with patch("k8s_charm.KubernetesServicePatch", lambda x, y: None):
+        with patch("charm.KubernetesServicePatch", lambda x, y: None):
             yield
 
 
 @pytest.fixture
 def charm_meta(substrate, charm_type) -> dict:
-    fname = {"lxd": "machine_metadata", "k8s": "k8s_metadata"}[substrate]
+    fname = {"k8s": "metadata"}[substrate]
```

Comment:
Feels like this entire function should probably be removed. Or at the very least rewritten as `return "metadata"`.

Reply:
#256